compiler/rustc_mir_transform/src/lower_intrinsics.rs

```diff
@@ -162,6 +163,30 @@ impl<'tcx> MirPass<'tcx> for LowerIntrinsics {
                             terminator.kind = TerminatorKind::Goto { target };
                         }
                     }
+                    sym::transmute => {
```

Comment:
This means the intrinsic can be removed from `interpret/intrinsics.rs`, right?

Reply:
Sure, can do.

(I left it in for cg_cranelift, since that seems to also still have copy_nonoverlapping in its intrinsics list.)

Reply:
cc @bjorn3 there seems to be some dead code in cg_cranelift

Reply:
Thanks for pointing this out. Fixed in bjorn3/rustc_codegen_cranelift@1ed6b05145a4729b5812e63aa15c0ccc7ec6495e.